src/javascripts/constants/message.js

```diff
@@ -14,5 +14,6 @@ export const ERROR_MESSAGE = {
 }
 
 export const ADD_TRANSACTION_MESSAGE = {
-    ADD_TRANSACTION_SUCCESS: "Add Transaction Succcess !"
+    ADD_TRANSACTION_SUCCESS: "Add Transaction Succcess !",
```

Comment:
Should be "Text content!" not "Text content !"

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!